lib/common/actions/index.js

```diff
@@ -131,6 +131,9 @@ function getErrorMessageFromJson (
     }
     // re-assign error message after it gets used in the detail.
     errorMessage = json.message || JSON.stringify(json)
+    if (json.detail.includes('conflicts with an existing trip id')) {
+      action = 'DEFAULT'
```

Comment:
Can we add a `console.warn` or similar letting us know that we did a slightly strange thing?